Source/SmallBasic.Editor/Libraries/Utilities/PredefinedColors.cs

```diff
@@ -11,7 +11,7 @@ namespace SmallBasic.Editor.Libraries.Utilities
 
     public static class PredefinedColors
     {
-        public const string TransparentHexColor = "#00000000";
+        public const string TransparentHexColor = "transparent";
```

Comment:
nit: can you rename to `Transparent`? It is no longer a hex color.
```suggestion
        public const string Transparent = "transparent";
```

Reply:
Done.